utils/req-res-log-validator.py

```diff
@@ -216,6 +218,9 @@ def process_file(docs, path):
             if res.error or res.queued:
                 continue
 
+            if req.command in IGNORED_COMMANDS:
+                continue
```

Comment:
can you explain why do we need this? we already skip them in missing_schema.

Reply:
we need to add both new commands in this set also to the case in `reqresAppendRequest`

perhaps it's worth dropping a comment in both py and c files that these lists should be in sync

Reply:
do these lists really need to be in sync? or are they there for different reasons?
why didn't we need this line in the validator till now?

Reply:
e.g. is it enough to add `sentinel|info-cache` to the list in `reqresAppendRequest` and then we don't need this new `IGNORED_COMMANDS` line in the validator?

Reply:
on the other hand, should we document the `sentinel info-cache` 's reply schema?
if we document it, then it can be removed from `IGNORED_COMMANDS `

in this case, i think we need to document it (this may require Guy's help, it's kind of hard for me to do it)

Reply:
IIUC the ignore list in reqresAppendRequest are commands for which is is not possible to create a reply schema (like DEBUG).
not sure what's the problem with info-cache though, and why we need to skip it in the new skip line you added.
waiting for someone to explain it to me.

Reply:
>do these lists really need to be in sync? or are they there for different reasons?
why didn't we need this line in the validator till now?

we need the list in the c code to ignore the "special" commands int eh req-res files
we need the list in python to avoid false positives on "missing schema" and "not hit" (which are based on COMMAND DOCS, where the "special" commands do appear)

both lists should be identical

>e.g. is it enough to add sentinel|info-cache to the list in reqresAppendRequest and then we don't need this new IGNORED_COMMANDS line in the validator?

correct

Reply:
and yes, it feels like SENTINEL INFO-CACHE should have a reply-schema

Reply:
ok, so SENTINEL INFO-CACHE is not like DEBUG, it should have a schema.
the list in the C file is to skip commands that cannot possibly have a schema (they can break the reqres file?).
and since we skip them in the C file, it causes them to be missing from the reqres files and then we have to skip them in the validator to silence the coverage warning?